chapters/creating_extension.md

```diff
@@ -117,19 +117,52 @@ For example, there is a list of `BuiltIn` enums, and its `PointSize` entry is:
   },
 ```
 
-An enum entry can also have an `extensions` attribute to list the SPIR-V extensions that enable the feature. As a design principle, we only use these on the capabilities introduced by the extension. Other tokens introduced by the extension are in turn guarded by the corresponding capability.
+Enum entries within a list should be ordered by their value. The spec generator writes table entries in the same order as JSON file order.
+
+An enum entry can also have attributes to describe when the feature exists:
+
+- The `version` attribute corresponds to the
+  "[missing before](https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#Unified)"
+  annotation in the SPIR-V specification.
+  If present, the `version` attribute indicates the first SPIR-V version in which the feature exists
+  as part of the core specification, without the need to use an `OpExtension` instruction.
+- The `extension` attribute, if present, indicates the name of the extension to use
+  with the `OpExtension` instruction when the feature is not yet part of the core SPIR-V specification.
+  The extension attribute takes the form of a list, in case a feature can be exposed through multiple
+  extensions.
+- If absent, the enumerant is available in all versions of SPIR-V
```

Comment:
If this is describing the spec, that's what absent means.

If describing the grammar, possibly refer to the README for SPIRV-Headers (https://github.com/KhronosGroup/SPIRV-Headers), which includes this:

### Reserving tokens in the JSON grammar

Care should be taken to follow existing precedent in populating the details of reserved tokens.
This includes:
- pointing to what extension has more information, when possible
- keeping enumerants in numeric order
- when there are aliases, listing the preferred spelling first
- adding the statement `"version" : "None"`

The only requirement would be to be consistent with this.